.eslintrc.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
+    env: {
+        browser: true,
+        commonjs: true,
+        es6: true,
+        node: true
+    },
+    extends: "eslint:recommended",
+    globals: {
+        Atomics: "readonly",
+        SharedArrayBuffer: "readonly",
+        // Remove after converting to ESM
+        define: "readonly"
+    },
+    overrides: [{
+      files: 'parser.js',
+      rules: {
+        // We could disable checking as auto-generated
```

Comment:
Yes just ignore the generated file(s).

Reply:
Ok, sure. I personally like to check, albeit without applying mere formatting rules, since one doesn't know if a dependency did its own linting, especially in such a case as this where the code to lint was generated at our calling, rather than being bundled with the dependency. But it was safe when checked. :)

IMHO, linting normal dependencies is also a good idea (though I haven't done that here no less because eslint does not, unforutnately imo, allow linting of `node_modules` even with ignore overrides). I think this would be even more compelling of a use case than checking one's own code to find not only bugs but unwanted privileged module/API use if the linting is robust enough (e.g., preventing or checking dynamic access to `require`, setting of `global`, etc.), but again this is not supported anyways.